klog_test.go

```diff
@@ -639,3 +639,73 @@ func TestInitFlags(t *testing.T) {
 		t.Fatal("Expected log_file_max_size to be 2048")
 	}
 }
+
+func createTestValueOfLoggingT() *loggingT {
+	l := new(loggingT)
+
```

Comment:
all the empty lines in this function can be removed.

Reply:
Thank you. I dealt with it.